crates/turbo-tasks-fs/src/lib.rs

```diff
@@ -264,6 +270,26 @@ impl DiskFileSystem {
         self.start_watching_internal(true)
     }
 
+    /**
+     * Create a watcher and start watching by creating `debounced` watcher
+     * via `full debouncer`
```

Comment:
I chose `full` debouncer impl compare to `mini` due to below ergonomics supported, which we can hope to avoid excessive event count for some cases.